test/utils.ts

```diff
@@ -10,7 +10,7 @@ import { createProgram } from "../factory/program";
 import { Config } from "../src/Config";
 import { SchemaGenerator } from "../src/SchemaGenerator";
 
-const validator = new Ajv();
+const validator = new Ajv({ allowUnionTypes: true });
```

Comment:
Shouldn't we instead fix the warning?

Reply:
I'm not actually sure what is causing the warnings specifically. There are another 4 the occur in addition to the multiple that this change prevents. I can dig into that perhaps in another PR

Reply:
Yes. I think I remember that ajv at some point started warning and we haven't looked into resolving the warnings yet. Hiding them isn't good since we will forget.

Reply:
I have removed the flag and will take a deeper look at all the warning when i get some more free time